src_cpp/opennnBridge/nerlWorkerOpenNN.cpp

```diff
@@ -285,6 +285,40 @@ namespace nerlnet
               break;
             }
          }
+         //------------ Distributed System Type ------------
+         switch (_distributed_system_type)
+         {
+            case WORKER_DISTRIBUTED_SYSTEM_TYPE_FEDCLIENTWEIGHTEDAVGCLASSIFICATION: // Federated Client Weighted Average Classification
+            {
+                // TODO Ori - Implement
+                // we need to update _train_labels_count
+                // look at number of output neurons
+                // if this is the first time resize the vector to the number of output neurons
+                // sum columns of labels (assert if num of labels not equal to num of output neurons)
+                int col_num = _data_set->get_columns_number();
+                std::shared_ptr<opennn::NeuralNetwork> neural_network_ptr = get_neural_network_ptr();
```

Comment:
You don't need to use the getter here, you can call directly to the object of the class. Please fix.